tools/releaseBuild/azureDevOps/templates/nuget.yml

```diff
@@ -39,11 +39,8 @@ jobs:
     displayName: 'Download PowerShell build artifacts'
     inputs:
       buildType: current
-      downloadType: specific
-      itemPattern: |
-        finalResults/PowerShell-*-win-*.zip
-        finalResults/powershell-*.tar.gz
-        results/powershell-*-osx-x64.tar.gz
+      downloadType: single
+      artifactName: finalResults
```

Comment:
Is `finalResults` a folder or a single file? Is this change actually copies all items in the `finalResults` folder to `$(System.ArtifactsDirectory)`?

Reply:
This change, yes downloads all files from finalResults.  I always hit some issue if I tried the filters, and this was faster than what we had before so I stopped here.

Reply:
You questions seems like you understood the change well enough.

Reply:
I saw `finalResults` appears in the build log as a folder :) A lucky guess